test/tests/orbit-common/unit/schema-test.js

```diff
@@ -130,7 +128,7 @@ test('#registerModel can register models after initialization', function(assert)
   assert.equal(schema.models['moon'], undefined, 'moon\'s definition has NOT been set');
 
   schema.on('modelRegistered', function(name) {
-    if (name = 'moon') {
+    if (name === 'moon') {
```

Comment:
That looks like a bug in the test that didn't turn out to affect the test. Your fix looks correct.